src/views/Auth/LoginPage/LoginPage.jsx

```diff
@@ -1,7 +1,157 @@
 import React from 'react';
+import styled from '@emotion/styled';
+import { Formik } from 'formik';
+import { Input } from '@/components/Input';
+import { Button } from '@/components/Button';
+import { FormError } from '@/components/FormError';
+import { ShortHeader } from '@/components/ShortHeader';
```

Comment:
```suggestion
import { ShortHeader } from '@/components/Header';
```

추가적으로 ShortHeader를 Left: 0을 적용 안하니 밀리는 현상이 발생하네요. 수정해놓겠습니다